include/service/entities/template_service.js

```diff
@@ -784,7 +825,7 @@ module.exports = function(pb) {
      * @return {string} The absolute path
      */
     TemplateService.getCustomPath = function(themeName, templateLocation){
-        return path.join(pb.config.docRoot, 'plugins', themeName, 'templates', templateLocation + '.html');
+        return CUSTOM_PATH_PREFIX + themeName + '/templates/' + templateLocation + '.html';
```

Comment:
This was another one that showed up on the profiler.  We spent a lot of time in this function.  Turns out that ```path.join``` is ~60 times slower than just concatenating the strings.